README.md

```diff
@@ -19,6 +19,29 @@ This repository contains:
 - a example of [a protocol based on the reproschema](./examples)
 - the [documentation](./docs)
 
+## Developing ReproSchema
+
+As of release 1.0.0, a linked data modeling language, [LinkML](https://linkml.io/linkml/), is used to create
+a [YAML file](linkml-schema/reproschema.yaml) with the schema.
+
+The [context file](contexts/reproschema) was automatically generated using LinkML,
+and then manually curated in order to support all the reproschema feature.
+
+### Release
+Upon release, there are additional formats, `jsonsld`, `turtle`, `n-triples`
```

Comment:
"Upon release, there are additional formats, `jsonsld`, `turtle`, `n-triples`
and `pydantic` that are created using `LinkML` tools, `reproschema-py`,
and reproschema-specific script for the `pydantic` format."
a bit confused by this sentence. `jsonsld`, `turtle`, `n-triples`
and `pydantic` are created from LinkML. and "`reproschema-py`,
and reproschema-specific script for the `pydantic` format." what is this part?